dockerfiles/slurm/Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM ubuntu:latest
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# Based on:
+#  - https://bitbucket.org/deardooley/agave-docker/src/master/test-containers/schedulers/slurm/Dockerfile
+#  - https://ubuntuforums.org/showthread.php?t=2404746
+#  - https://stackoverflow.com/questions/46966876/installing-emulating-slurm-on-an-ubuntu-16-04-desktop-slurmd-fails-to-start
+
+RUN apt-get update && apt-get install -y \
+    at \
+    lsof \
+    munge \
+    slurm-wlm \
+    supervisor \
+    python3.7 \
+    python3-pip \
+    python3.7-dev \
```

Comment:
Installing both 3.7 and 3.7-dev?

Reply:
It was missing headers for... pyzmq or protobuf? I think of of these modules failed. Initially I installed only python3.7. Then noticed it was missing pip, then added it. And kept adding modules as the docker image failed to successfully build & run